apps/system/js/statusbar.js

```diff
@@ -365,14 +366,19 @@ var StatusBar = {
   },
 
   setAppearance: function() {
-    var app = Service.currentApp;
-
     // The statusbar is always maximised when the phone is locked.
     if (this._inLockScreenMode) {
       this.element.classList.add('maximized');
       return;
     }
 
+    var app = Service.query('getTopMostWindow');
+    // In some cases, like when opening an app from the task manager, there
+    // temporarily is no top most window, so we cannot set an appearance.
+    if (!app) {
+      return;
+    }
+
     // Fetch top-most window to figure out color theming.
     var topWindow = app.getTopMostWindow();
```

Comment:
Yeah I wasn't sure since one `getTopMostWindow` comes from the hierarchy manager, and the other comes from the app window. I left it alone just to be sure.